lib/Client.php

```diff
@@ -46,9 +46,9 @@ public function readConfig($path = null, $output = false)
             }
         }
 
-        $this->apiKey = $conf['apiKey'] ?: $this->apiKey;
-        $this->apiVersion = $conf['apiVersion'] ?: $this->apiVersion;
-        $this->environment = $conf['environment'] ?: $this->environment;
+        $this->apiKey = isset($conf['apiKey']) ? $conf['apiKey'] : $this->apiKey;
```

Comment:
All these `isset` checks are basically the same. I don't understand why yet, but my PHP version/configuration simply returns `null` if the key isn't defined. For others, it seems to throw an exception, which I haven't been able to reproduce. 

The null coalesce `??` operator (`$conf['apiKey'] ?? $this->apiKey`) is so much cleaner, but only available in PHP 7+.

Reply:
that and the `?:` are so nice. new since i last used PHP ;)